Cargo.toml

```diff
@@ -0,0 +1,19 @@
+# This file is part of Astarte.
+#
+# Copyright 2024 SECO Mind Srl
+#
+# SPDX-License-Identifier: Apache-2.0
+
+[package]
+name = "stream-rust-test"
+version = "0.1.0"
+edition = "2021"
+license = "Apache-2.0"
+homepage = "https://astarte.cloud/"
+keywords = ["rust", "iot", "astarte", "mqtt-client"]
+readme = "README.md"
+repository = "https://github.com/astarte-platform/stream-rust-test"
+rust-version = "1.72.0"
+description = "Astarte Rust SDK based data stream test."
```

Comment:
Try running `cargo clippy -- -D clippy::cargo` to check the missing fields (e.g. `categories`)